src/libs/Navigation/linkTo.js

```diff
@@ -5,6 +5,43 @@ import linkingConfig from './linkingConfig';
 import getTopmostReportId from './getTopmostReportId';
 import getStateFromPath from './getStateFromPath';
 
+/**
+ * Motivation for this function is described in NAVIGATION.md
+ *
+ * @param {Object} action action generated by getActionFromState
+ * @param {Object} state The root state
+ * @returns {{minimalAction: Object, targetName: String}} minimal action is the action that we should dispatch and targetName is the name of the target navigator.
+ * targetName name is necessary to determine if we are going to use REPLACE for navigating between RHP flows.
+ */
+function getMinimalAction(action, state) {
+    let currentAction = action;
+    let currentState = state;
+    let currentTargetKey = null;
+    let targetName = null;
```

Comment:
This PR caused a regression, for more details refer this [PR](https://github.com/Expensify/App/pull/22355)